poc_mobile_verifier/src/speedtests.rs

```diff
@@ -295,12 +301,7 @@ where
                 latency_avg_ms: sum_latency / window_size as u32,
             }
         } else {
-            Average {
-                window_size,
-                upload_speed_avg_bps: 0,
-                download_speed_avg_bps: 0,
-                latency_avg_ms: 0,
-            }
+            Default::default()
```

Comment:
NIT: could we make this `Average::default()` so it is clearer?

Reply:
Absolutely; agreed

Reply:
fixed [50a7d3e](https://github.com/helium/oracles/pull/158/commits/50a7d3efba826fdadbce3633e46bdf917c59b876)